src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int MAX_VALUE = Color.values().length;
+
+    public String getColorOfFigure() {
+        final int indexOfColor = new Random().nextInt(MAX_VALUE);
+        final Color colorOfFigure = Color.values()[indexOfColor];
```

Comment:
Why colorOfFigure ? I do not see any Figures here.

Reply:
not fixed
```suggestion
        final Color color = Color.values()[index];
```